app/src/components/market/sections/market_profile/market_status/open.tsx

```diff
@@ -132,11 +132,11 @@ const Wrapper = (props: Props) => {
     <ViewCard>
       <MarketTopDetails marketMakerData={marketMakerData} toggleTitle="Pool Information" />
       {renderTableData()}
-      {isQuestionOpen && IS_CORONA_VERSION && openQuestionMessage}
+      {isQuestionOpen && SHOW_ANSWER_IN_REALITIO && openQuestionMessage}
       <WhenConnected>
         <ButtonContainer>
-          {!IS_CORONA_VERSION && poolButton}
-          {isQuestionOpen ? (IS_CORONA_VERSION ? null : openInRealitioButton) : buySellButtons}
+          {SHOW_POOLING_BTN && poolButton}
+          {isQuestionOpen ? (SHOW_ANSWER_IN_REALITIO ? null : openInRealitioButton) : buySellButtons}
```

Comment:
Maybe we need 3 variables here, and more expressive names, but I wasn't sure about why these buttons were hidden.